setup.py

```diff
@@ -39,14 +39,10 @@
     "pytest-cov==2.7.1",
     "pytest-helpers-namespace==2019.1.8",
     'pytest==3.10.1',
-    'gitdb2==2.0.6',
-    "GitPython==2.1.14",
 ]
 
 # These are not direct dependencies, but we need to lock down their versions here
 EXTRA_REQ = [
```

Comment:
Looks like this is list is empty; is there any reason to not just remove it?

Reply:
A flawed expectation that it will probably be needed again, but that's really not a good enough reason to keep it around.